crates/matrix-sdk-sqlite/src/utils.rs

```diff
@@ -255,6 +241,32 @@ impl<'a> SqliteTransactionExt for Transaction<'a> {
     }
 }
 
+/// Extension trait for a [`rusqlite::Connection`] that contains a key-value
+/// table named `kv`.
+///
+/// The table should be created like this:
+///
+/// ```sql
+/// CREATE TABLE "kv" (
+///     "key" TEXT PRIMARY KEY NOT NULL,
+///     "value" BLOB NOT NULL
+/// );
+/// ```
+pub(crate) trait SqliteKeyValueStoreConnectionExt {
```

Comment:
I would rename to `SqliteKeyValueStoreConnExt`, so that we use `Conn` everywhere in then names and it feels more consistent. What do you think?

Reply:
It also looks very similar to `SqliteKeyValueStoreAsyncConnExt` this way, though I don't know if there are supposed to represent the same thing.

Reply:
They do basically represent the same thing, a trait for methods to interact with the `kv` store. But one is a trait with blocking methods for `rusqlite::Connection`, and the other is a trait with async methods for `SqliteAsyncConn`, which is why it's the only part of the name that changes.

Reply:
ok so can you rename to `SqliteKeyValueStoreConnExt` (`Connection` becomes `Conn`)? What do you think?

Reply:
Sure

Reply:
Done